.github/workflows/release-npm.yml

```
@@ -14,10 +14,10 @@ jobs:
       - uses: mymindstorm/setup-emsdk@v14
         with:
           # See https://github.com/tree-sitter/tree-sitter/blob/master/cli/emscripten-version
```

[Previously pointed to the version](https://github.com/tree-sitter/tree-sitter/blob/v0.20.2/cli/emscripten-version), though [link is now dead](https://github.com/tree-sitter/tree-sitter/blob/master/cli/emscripten-version) so we may want to [update and use a permalink](https://github.com/tree-sitter/tree-sitter/blob/cd3d967f3cc1ab005e6f122f2cd034d0b02321ff/cli/loader/emscripten-version) - similarly applying to [reference in testworkflow.yml](https://github.com/cucumber/language-service/pull/235/files#diff-b1d8ed23b728461de1cd6c65454bb692f69d5e4de0a3e32f38e0ddec13103853R32)

---

I'd suggest the following:
```suggestion
          # See https://github.com/tree-sitter/tree-sitter/blob/v0.24.6/cli/loader/emscripten-version
          # (choose the Git tag that corresponds to the version of web-tree-sitter specified in package.json)
          # Should be kept in sync with .github/workflows/test-javascript.yml
```